package.json

```diff
@@ -3,9 +3,7 @@
   "description": "The self-hosted Web IRC client",
   "version": "4.4.3",
   "preferGlobal": true,
-  "bin": {
-    "thelounge": "index.js"
-  },
+  "bin": "index.js",
```

Comment:
?

Reply:
Ah that's some other unwelcome jackassery that yarn is performing. Hadn't intended to commit it (been doing  add -p to avoid it). Will back out out

Reply:
this is still there